build-tools/automation/templates/core-tests.yaml

```diff
@@ -64,7 +64,7 @@ steps:
   condition: eq('${{ parameters.runNativeTests }}', 'true')
   inputs:
     command: test
-    arguments: bin/Test$(Build.Configuration)/Java.Interop-Tests.dll
+    arguments: bin/Test$(Build.Configuration)$(NetCoreTargetFrameworkPathSuffix)/Java.Interop-Tests.dll
```

Comment:
Do we want to run both versions of these tests?

Reply:
This has been addressed in the latest force-push.  Now both versions are run.